spec/helpers/visit_helper_spec.rb

```diff
@@ -84,7 +84,9 @@ def visit
     end
 
     it "provides a formatted date for when a response may be sent out" do
-      expect(helper.when_to_expect_reply(Date.parse("2014-10-03"))).to eq("Monday  6 October")
+      Timecop.travel(Date.parse("2014-10-03")) do
+        expect(helper.when_to_expect_reply).to eq("Friday  3 October")
```

Comment:
This is because the Rails date helper places a white space where the tens-digit would be for single digit numbers. Which is rubbish. I did put `gsub` fixes into views to get round this. Which I'm assuming has been missed (or removed) for this particular view, judging by this spec.